projects/ADuCM3029_demo_cn0503/src/platform_source/flash.c

```diff
@@ -148,7 +148,7 @@ int32_t flash_write(struct flash_dev *dev, uint32_t flash_addr,
 	int32_t ret;
 
 	do {
-		temp = (FLASH_PAGE_SIZE_BYTES / 4) - (flash_addr & 0x7FF);
+		temp = (FLASH_PAGE_SIZE_BYTES - (flash_addr & 0x7FF)) / 4;

```

Comment:
Can we have these changes in a separate commit as well? It would be much easier to integrate them outside of the project if needed.